cardinal/plugin_persona.go

```diff
@@ -19,7 +19,17 @@ import (
 	"pkg.world.dev/world-engine/cardinal/types/engine"
 )
 
-var _ Plugin = (*personaPlugin)(nil)
+var (
+	_ Plugin = (*personaPlugin)(nil)
+
+	// globalPersonaTagToAddressIndex keeps track of the mapping of persona-tags->signer-address so it doesn't need to
+	// be recomputed each tick.
+	globalPersonaTagToAddressIndex personaIndex
+	// The tick that the globalPersonaTagToAddressIndex was built on. In normal usage, wCtx.CurrentTick should always
+	// be greater than this number, but during tests the currentTick will be reset. Tracking this number at the global
+	// is easier than updating each test to reset this global value.
+	tickOfPersonaTagToAddressIndex uint64
+)
```

Comment:
Globals are bad, but this is a temp hack. It should be mentioned in the comments.

Reply:
I added a TODO and linked to the linear ticket related to indexing.